css/temp.css

```diff
@@ -47,3 +47,10 @@
   margin-top: 20px;
   margin-bottom: 20px;
 }
+
+
+/* Apply solid white bg to images with text (on a transparent bg)
+so that night mode doesn't garble text on invert to gray */
+.white-bg {
```

Comment:
Does it make sense for us to keep this one? If we're just trying to solve the night mode issue, maybe we don't need it anymore.